content/docs/instrumenting/exposition_formats.md

```diff
@@ -5,104 +5,98 @@ sort_rank: 6
-characters (after the metric name), but the backslash and the line-feed
+characters (after the metric name), but the backslash and the line feed
 characters have to be escaped as `\\` and `\n`, respectively. Only one `HELP`
-line may exist for the same metric name.
+line may exist for any given metric name.
 
 If the token is `TYPE`, exactly two more tokens are expected. The first is the
 metric name, and the second is either `counter`, `gauge`, `histogram`,
 `summary`, or `untyped`, defining the type for the metric of that name. Only
-one `TYPE` line may exist for the same metric name. The `TYPE` line for a
-metric name has to appear before the first sample is reported for that metric
+one `TYPE` line may exist for a given metric name. The `TYPE` line for a
+metric name must appear before the first sample is reported for that metric
 name. If there is no `TYPE` line for a metric name, the type is set to
-`untyped`. Remaining lines describe samples, one per line, with the following
-syntax (EBNF):
+`untyped`.
+
+The remaining lines describe samples (one per line) using the following syntax
+([EBNF](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form)):
 
-    metric_name [
-      "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
-    ] value [ timestamp ]
+```
+metric_name [
+  "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
+] value [ timestamp ]
+```
 
-`metric_name` and `label_name` have the usual Prometheus expression language restrictions. `label_value` can be any sequence of UTF-8 characters, but the backslash, the double-quote, and the line-feed characters have to be escaped as `\\`, `\"`, and `\n`, respectively.
-`value` is a float, and timestamp an `int64` (milliseconds since epoch, i.e. 1970-01-01 00:00:00 UTC, excluding leap seconds), represented as required by the [Go strconv package](http://golang.org/pkg/strconv/) (see functions `ParseInt` and `ParseFloat`). In particular, `Nan`, `+Inf`, and `-Inf` are valid values.
+In the sample syntax:
 
-All lines for a given metric must be provided as one uninterrupted group, with
+*  `metric_name` and `label_name` carry the usual Prometheus expression language restrictions.
+* `label_value` can be any sequence of UTF-8 characters, but the backslash (`\`, double-quote (`"`}, and line feed (`\n`) characters have to be escaped as `\\`, `\"`, and `\n`, respectively.
+* `value` is a float.
+* the `timestamp` is an `int64` (milliseconds since epoch, i.e. 1970-01-01 00:00:00 UTC, excluding leap seconds), represented as required by the [Go strconv package](http://golang.org/pkg/strconv/) (see functions [`ParseInt`](https://golang.org/pkg/strconv/#ParseInt) and [`ParseFloat`](https://golang.org/pkg/strconv/#ParseFloat)). In addition to standard integers, `Nan`, `+Inf`, and `-Inf` are valid values representing not a number, positive infinity, and negative infinity, respectively.
```

Comment:
This comment still applies: https://github.com/prometheus/docs/pull/1083/commits/d6e75b7158d5d216c933354e9797c7b028af7b75#r198996763

The last sentence is about sample values and the `ParseFloat` bit of the previous sentence too (since sample values are floats, timestamps are ints).